tsl/test/expected/cagg_on_cagg.out

```diff
@@ -18,22 +18,33 @@
 \set BUCKET_WIDTH_1ST 'INTEGER \'1\''
 \set BUCKET_WIDTH_2TH 'INTEGER \'5\''
 \set BUCKET_WIDTH_3TH 'INTEGER \'10\''
-\ir include/cagg_on_cagg_common.sql
+-- Different order of time dimension in raw ht
+\set IS_DEFAULT_COLUMN_ORDER FALSE
+\ir include/cagg_on_cagg_setup.sql
 -- This file and its contents are licensed under the Timescale License.
 -- Please see the included NOTICE for copyright information and
 -- LICENSE-TIMESCALE for a copy of the license.
+-- CAGGs on CAGGs tests
 \if :IS_DISTRIBUTED
 \echo 'Running distributed hypertable tests'
 \else
 \echo 'Running local hypertable tests'
 Running local hypertable tests
 \endif
 SET ROLE :ROLE_DEFAULT_PERM_USER;
--- CAGGs on CAGGs tests
-CREATE TABLE conditions (
-  time :TIME_DIMENSION_DATATYPE NOT NULL,
-  temperature NUMERIC
-);
+DROP TABLE IF EXISTS conditions CASCADE;
+psql:include/cagg_on_cagg_setup.sql:14: NOTICE:  table "conditions" does not exist, skipping
```

Comment:
Is this necessary?

Reply:
At this point no but the new file `include/cagg_on_cagg_setup.sql` will be included several times so we'll need to cleanup the hypertable to re-setup the test case